### ISSUE TITLE
Issues building when trying the Make, CMake Templates.

### ISSUE THREAD

**Issue by KnockerPulsar** [RobLoach/raylib-cpp#120]
I've been toying with Raylib for a couple of days now but as someone used to C++ I wanted to see if I can get comfy with raylib-cpp. I've been using Raylib with VSCode on Windows 10 if that's any help. I have been mostly doing small college projects so I haven't needed to use CMake or Make before.

I've been trying with both the Make and CMake templates to no avail.
For the CMake template, everything is fine until the "make" step, when I enter this command into either PowerShell or cmd it returns:
**"make: *** No targets specified and no makefile found.  Stop."**
I found a .sln file inside the build folder so I decided to fire it up and build it. That at least works.

For the Make template, I followed the steps and made sure I have the dependencies needed for Windows. After entering "mingw32-make setup" it seems that it encounters a bunch of errors but ignores them. Entering "mingw32-make" after that causes the compiler to throw a fatal error:
**"fatal error: opening dependency file bin/main.d: No such file or directory
 }
 ^
compilation terminated.
mingw32-make: *** [Makefile:88: bin/main.o] Error 1"**
This is the output of the powershell in more detail : [makeLogThingy.txt](https://github.com/RobLoach/raylib-cpp/files/5992699/makeLogThingy.txt)

Trying the code given in the "Compiling" subsection of the repo's README also yields a similar result to the CMake template.

**Comment by RobLoach** [comment]
This is a header-only library so it should require including it and that's all. If you get raylib to work in your project, you should be able to include the raylib-cpp.hpp in your main.cpp file and be good to go.

**Comment by KnockerPulsar** [comment]
Ok, I tried that before but it caused the program to close right after opening for no reason..
Anyway, the code seems up and running for now.
For other Make/CMake noobs out there, I got my code to run on VSCode doing the following:
1. Go into the Raylib repo/projects and download "VSCode".
2. Extract the Makefile and .vscode folder where your cpp is
3. Go into the Makefile, search for C++, you'll find a couple of comments telling you to define which compiler for which language. Below these comments, you'll find "CC=gcc" change "gcc" to "g++"
4. Search for "Define compiler flags", you'll find a comment block explaining the compiler flags, go down to "CFLAGS += blah blah blah", you'll find "-std=C99" change that to "-std=c++17"
5. Go into the .vscode folder you just extracted, open "tasks.json", under the windows section you'll find "args", it'll contain a string similar to "OBJS=${fileBasenameNoExtension}.c", change the ".c" into ".cpp"
6. Don't forget to include raylib-cpp.hpp

This should get code compiling and running, I'm pretty sure this isn't the most optimal way to do things but it might be a good starting point to anyone struggling with getting their code running.

Thanks Rob!
